### PR TITLE
Updated getImageData() and LoadImages() in accordance with API version

### LINKED ISSUE #149: API v1: update getImageData() and LoadImages()
[getImageData(id) in infoPopupModule.js](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/infoPopupModule.js#L163) and [loadImages() in bookmarkModule.js](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/bookmarkModule.js#L82) uses the image search endpoint of the cc catalog API ([old docs](https://api.creativecommons.engineering/v0/#operation/image_detail) and [new doc](https://api.creativecommons.engineering/v1/#operation/image_detail)). Update and mould the implementation of these function in accordance of the new API version.
- change the url from http://api.creativecommons.engineering/image/{id} to https://api.creativecommons.engineering/v1/images/{id}
- Since `provider_code` key is removed in the new version, use `source` in loadImages().
- Since `provider` key is removed in the new version, use `source` in getImageData().
- Also, change the instances of `provider` variable to `source` in these functions.

### PR DESCRIPTION
**Pull request by Arunhari333**
Fixes #149 

**Description**

Updated `getImageData()` and `LoadImages()` by replacing the URL in accordance with the new API version and changed instances of `provider` and `provider_code` to `source`.

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

**Comment by Arunhari333** [comment]
I don't really understand the problem, all I changed was the names. Could you point out my mistake and help me correct it?

**Comment by Arunhari333** [comment]
Very sorry, did not see that error among the other unimportant ones like " 'require' is not defined ". I will fix it tomorrow itself.

File: src/popup/searchModule.js [hunk 1]
```diff
@@ -36,13 +36,13 @@ export function getRequestUrl(
...
-  userSelectedProvidersList,
+  userSelectedSourcesList,
   page,
 ) {
   if (userSelectedUseCaseList.length > 0) {
-    return `https://api.creativecommons.engineering/image/search?q=${inputText}&page=${page}&pagesize=20&lt=${userSelectedUseCaseList}&provider=${userSelectedProvidersList}`;
+    return `https://api.creativecommons.engineering/v1/images?q=${inputText}&page=${page}&page_size=20&license_type=${userSelectedUseCaseList}&source=${userSelectedSourcesList}`;
   }
-  return `https://api.creativecommons.engineering/image/search?q=${inputText}&page=${page}&pagesize=20&li=${userSelectedLicensesList}&provider=${userSelectedProvidersList}`;
+  return `https://api.creativecommons.engineering/v1/images?q=${inputText}&page=${page}&page_size=20&license=${userSelectedLicensesList}&source=${userSelectedSourcesList}`;
```
Comment (makkoncept) [inline src/popup/searchModule.js]: You have not pulled the changes. You yourself resolved this issue in https://github.com/creativecommons/ccsearch-browser-extension/pull/154.

**Review by makkoncept — CHANGES_REQUESTED** [review]

**Comment by Arunhari333** [comment]
Sorry, messed up while using rebase. Its Ok now right?

**Review by makkoncept — APPROVED** [review]
Looks good!

**Pull request merged**